midonet-tools/src/share/prepare-java

```diff
@@ -17,19 +17,14 @@
 # The first existing directory is used for JAVA_HOME if needed.
 LIB_PATTERNS='{lib64,lib}'
 JVM_SEARCH_DIRS=$( eval echo \
-                 /usr/$LIB_PATTERNS/jvm/java-1.8.0-openjdk-amd64 \
-                 /usr/$LIB_PATTERNS/jvm/java-8-openjdk-amd64 \
-                 /usr/$LIB_PATTERNS/jvm/java-8-oracle \
-                 /usr/$LIB_PATTERNS/jvm/zulu-8-amd64 \
-                 /usr/$LIB_PATTERNS/jvm/jre-1.8.0-openjdk \
-                 /usr/$LIB_PATTERNS/jvm/jre-8-oracle-x64 \
-                 /usr/$LIB_PATTERNS/jvm/jdk-8-oracle-x64)
+                 /usr/$LIB_PATTERNS/jvm/adoptopenjdk-11-hotspot-amd64 \
+                 )
 
 # zulu prints "openjdk version" while openjdk and oracle print "java version"
 check_for_java8() {
```

Comment:
is this now actually checking for java 11?

Reply:
Fixed